runtime/src/utils.rs

```diff
@@ -19,15 +20,16 @@ use sp_std::mem::size_of;
 pub const ENCODED_EXTRINSIC_SIGNATURE_LENGTH: u64 = 102;
 
 /// Compute total fee for executing a call
-pub fn compute_fee(call: Call) -> Balance {
+pub fn compute_fee(call: RuntimeCall) -> Balance {
     let xt = UncheckedExtrinsic::new_unsigned(call);
     let length = xt.encode().len() as u64 + ENCODED_EXTRINSIC_SIGNATURE_LENGTH;
     let dispatch_info = &<UncheckedExtrinsic as GetDispatchInfo>::get_dispatch_info(&xt);
     let weight_fee = <Runtime as pallet_transaction_payment::Config>::WeightToFee::weight_to_fee(
         &dispatch_info.weight,
     );
-    let len_fee =
-        <Runtime as pallet_transaction_payment::Config>::LengthToFee::weight_to_fee(&length);
+    let len_fee = <Runtime as pallet_transaction_payment::Config>::LengthToFee::weight_to_fee(
+        &Weight::from_ref_time(length),
```

Comment:
```suggestion
        &Weight::from_parts(length, 0u64),
```
I did some research and this function will be deprecated: https://paritytech.github.io/substrate/master/sp_weights/struct.Weight.html#method.from_ref_time
The suggestion is what they recommend

Reply:
Updated in 51fb7635553d2aaedb59b6fc94412d244b233851